src/odemis/model/__init__.py

```
@@ -31,6 +31,7 @@
 from ._core import *
 from ._metadata import *
 from ._dataio import *
+from ._constants import *
```

Ruff (0.15.21)

[error] `from ._constants import *` used; unable to detect undefined names

(F403)